prefect_azure/container_instance.py

```diff
@@ -543,7 +649,7 @@ def _get_logs(
         client: ContainerInstanceManagementClient,
         container_group: ContainerGroup,
         max_lines: int = 100,
-    ) -> str:
+    ) -> Union[str, None]:
```

Comment:
If I'm not mistaken:
```suggestion
    ) -> str:
```